### ISSUE TITLE
PC is not connecting to ESP

### ISSUE THREAD

**Issue by arvid888** [scottlawsonbc/audio-reactive-led-strip#138]
I've uploaded the Arduino code to a nodeMCU, plugged it in and started the visualization program. The program shows the GUI correctly but the LEDs won't light up. 😞 

I've tried to use the nodeMCU offline to the led's and it works...

Might be an error in the Arduino code although I've set it up correctly.

 Maybe I should try to turn on the python program first, which way do you guys do it?

Thanks.

**Comment by ItsTheJetstream** [comment]
Try using the serial monitor. In the standard Arduino program you can open it and select the COM port of your ESP. You might have to fiddle around with the baud rate, just change it, press reset on the ESP and look at the printet lines in the monitor. When everything is installed correctly you should see the outputs of the code.

**Comment by arvid888** [comment]
I’m seeing an fps counter that shows constant 0’s on the serial monitor aswell as that it connected to wifi, the light lights up only 2 leds, one green and one white. also tried to load up a Fastled  test and it gives me same error with 2 leds lighting up but with the neopixel library everything works...
Something wrong with the firmware of the nodeMCU?

**Comment by caioseda** [comment]
i'm with the same problem, i think that could be that the esp didn't connected properly with wifi

**Comment by PekashuxD** [comment]
I have the same problem, did someone solve it?

**Comment by caioseda** [comment]
@PekashuxD Yes! It was a connections issue. I've opened serial monitor and discovered that the ESP was connecting to wifi with a different IP that was coded. Opened Wirelles client's on my router and got the MAC address of ESP and made the IP Static (with the IP that was already connected) so with that i changed the IP in config.py and in .ino to make sure that everything is equivalent. As soon as i run visualization.py got some dope RGB in my room :D

**Comment by PekashuxD** [comment]
It appears to me in the arduino ide fps = 0, while in python if it sends me the fps :(

![Captura](https://user-images.githubusercontent.com/59328993/71566991-e93a0000-2a89-11ea-973b-052449fc4c2c.JPG)
![Captura2](https://user-images.githubusercontent.com/59328993/71566992-eb03c380-2a89-11ea-8423-ef5f7d33abd5.JPG)
